app/pages/api/auth/[...nextauth].ts

```diff
@@ -138,5 +138,9 @@ export default NextAuth({
     secret: process.env.SECRET,
   },
   debug: true,
-  theme: "auto",
+  theme: {
+    color: "auto",
+    logo: "https://crm.newtelco.de/images/logo_newtelco.png",
+    brandColor: "#67B246"
```

Comment:
I'm fine to leave it like this, but maybe adding a NextAuth.js logo/color would make more sense? 😜

Reply:
Oops my bad! Didn't mean to keep those test values of mine in there :facepalm: